conda_forge_tick/auto_tick.py

```diff
@@ -546,6 +548,8 @@ def migration_factory(
                 set(loaded_yaml) | {ly.replace("_", "-") for ly in loaded_yaml}
             ) & all_package_names
         package_names = package_names - excluded_feedstocks
+        if "python" in package_names:
+            package_names.add("pypy-meta")
```

Comment:
What happens without this?

Reply:
or maybe, shoudl modify the migrator instead?

Reply:
actually looks like we don't need it, graph looks the same